pkg/odo/cli/url/describe.go

```diff
@@ -0,0 +1,112 @@
+package url
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/openshift/odo/pkg/config"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/openshift/odo/pkg/odo/util"
+	"github.com/openshift/odo/pkg/odo/util/completion"
+	"github.com/openshift/odo/pkg/url"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
+)
+
+// DescribeRecommendedCommandName is the recommended describe command name
+const describeRecommendedCommandName = "describe"
+
+var describeExample = ktemplates.Examples(`  # Describe a URL
+%[1]s myurl
+`)
+
+// URLListOptions encapsulates the options for the odo url list command
+type URLDescribeOptions struct {
+	localConfigInfo  *config.LocalConfigInfo
+	componentContext string
+	url string
+	*genericclioptions.Context
+}
+
+// NewURLDescribeOptions creates a new URLCreateOptions instance
+func NewURLDescribeOptions() *URLDescribeOptions {
+	return &URLDescribeOptions{&config.LocalConfigInfo{}, "", "", &genericclioptions.Context{}}
+}
+
+// Complete completes URLDescribeOptions after they've been Listed
+func (o *URLDescribeOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	o.Context = genericclioptions.NewContext(cmd)
+	o.localConfigInfo, err = config.NewLocalConfigInfo(o.componentContext)
+	if err != nil {
+		return errors.Wrap(err, "failed intiating local config")
+	}
+	if len(args) == 1 {
```

Comment:
if we implement https://github.com/openshift/odo/pull/2694/files#r391425980, we won't need this check

Reply:
ok, will do